perfkitbenchmarker/linux_benchmarks/cloudsuite_inmemory_analytics_benchmark.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
```

Comment:
I suggest making the file name cloudsuite_in_memory_analytics_benchmark.py to be consistent about the 'in-memory-analytics' name. It looks like we didn't do that with 'web-search' -> 'websearch', but we did do it with 'web-serving' -> 'web_serving'.